.travis.yml

```diff
@@ -1,19 +1,41 @@
 language: python
-
 cache: pip
+env:
+  - PR_AUTHOR=${TRAVIS_PULL_REQUEST_SLUG::-15}
```

Comment:
Allows us to skip the automerge build if the PR_AUTHOR is not `openshift-cherrypick-robot`. Check line 36
- fails silently https://app.travis-ci.com/github/openshift/openshift-docs/builds/245020194 VS https://app.travis-ci.com/github/openshift/openshift-docs/builds/245019610